types/office-js/index.d.ts

```diff
@@ -330,7 +330,37 @@ declare namespace Office {
      * @param useShortNamespace True to use the shortcut alias; otherwise false to disable it. The default is true.
      */
     function useShortNamespace(useShortNamespace: boolean): void;
+    /**
+     * Represents the add-in.
+     */
+    const addin: Addin;
```

Comment:
@Rick-Kirkham Do you have context on these office-js changes? Thx.

Reply:
@ElizabethSamuel-MSFT It's about enabling custom keyboard shortcuts in Office Add-ins. An action is basically a function that runs when a key combination, like Ctrl-Alt-Z is pressed.

Reply:
@Rick-Kirkham, I think @ElizabethSamuel-MSFT is asking about the SharedRuntime API change. It is for the bug you created as office.addin deifnition is not available in office-js.